### PR TITLE
Adds units (rads) to geiger counter readout

### PR DESCRIPTION
**Pull request by silverplatedelta**
## What Does This PR Do
Adds units (rads) to the chat results of the geiger counter.

## Why It's Good For The Game
Quantifying numbers with units is good

## Testing
Disable grav gen, confirm units are present. Confirm emag functionality is untouched.

## Changelog
:cl:
tweak: Adds units (rads) to geiger counter readout
/:cl:

### CODE REVIEW

**Comment by CinnamonSnowball** [comment]
As far as I'm aware, whatever is the measure of radioactive contamination is not the same for radiation (*being* irradiated), if they're even related at all; presumably they shouldn't use the same units?

**Comment by silverplatedelta** [comment]
> As far as I'm aware, whatever is the measure of radioactive contamination is not the same for radiation (_being_ irradiated), if they're even related at all; presumably they shouldn't use the same units?

Both yes and no. If you don't mind a quick nerding out, my understanding of these units boils down to: 
a rad is a measure of absorbed radiological dose, in reference to an organism. These are the units we use in game.
A grey is also a mesaure of absorbed radiological dose equal to 100 rads
A seivert is the unit of radiological dose that takes into account what kind of radiation an individual was exposed too. 
In modern medicine, we use Sieverts to calculate radiological dosage, but this does not carry over well to spess game where alpha, beta, gamma, and neutron decay are all treated the same. 
Rads have been popularized by games like the fallout franchise, so their use remains. 

to measure the envriomental radiation, units like the rotengen are used (if you watched HBO's chernobyl, you will be familiar). This is an mesaurement of the ionizing electromagnetic radiation (only x-rays and gamma rays, alpha and beta radiation are not accounted for) in a given air mass. 

Ideally, for the radiation of an object, we would be using becquerels, which is the SI unit for contamination, and is defined by the amount of decays per nucleus per second. 

TLDR:
The problem we run into, is that the amount of decay events per nucleus per second and absorbed dosage are not compatible with our games single value for radiation. To keep things simple for the playerbase, the unit popularized by videogames, the rad is used. I would not be opposed to a total radiation rework, but I lack the technical knowledge to go about doing so.

**Comment by CinnamonSnowball** [comment]
I moreso meant like, you can get hundreds of units of irradiation from exposure to only a couple dozen units of radioactive contamination
So unless I'm lacking some knowledge about how radiation works, labeling them as the same thing would just be confusing (and people *already* confuse irradiation vs. radioactivity)

**Review by SteelSlayer — APPROVED** [review]

**Comment by silverplatedelta** [comment]
> I moreso meant like, you can get hundreds of units of irradiation from exposure to only a couple dozen units of radioactive contamination So unless I'm lacking some knowledge about how radiation works, labeling them as the same thing would just be confusing (and people _already_ confuse irradiation vs. radioactivity)

Moreso the otherway around from my undestanding, hundreds of rotengen per hour can add up to 1-2 seiverts quite quickly. However the game uses a single unit for all radiation related things. While I would like this to be reflected properly in the future, the goal of this PR was to fix a minor typographical issue.

**Review by Sirryan2002 — APPROVED** [review]
